src/02-common-principles.md

```diff
@@ -55,6 +56,39 @@ misunderstanding we clarify them here.
     due to different subject response or randomized nature of the stimuli). Run
     is a synonym of a data acquisition.
 
+## File names consist of key-value pairs (entities)
+
+We define file names with a chain of key-value pairs, each of which we call
```

Comment:
The "We" and "you" that show up a bit here feel off relative to most of the document. I think we generally use the passive voice or imperative mood. If it would be useful to have a native English speaker do a sweep through to uniformize the voice, I can do that.

Reply:
That'd be great :+1:

Reply:
See sappelhoff#2.